spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc

```diff
@@ -105,6 +105,10 @@ exchanges).
 |Displays arbitrary application info.
 |Yes
 
+|`integrationgraph`
+|Exposes the Spring Integration graph.
+|No
```

Comment:
@snicoll Can you update this in your polishing commit to indicate that it will be enabled by default? Thanks!

Reply:
Yup, thanks for the ping.